### PR TITLE
Dev-US11 merge

### PR DESCRIPTION
**Pull request by CKindred**
This is my merge of US11 and the dev-branch. Please have a look and see if you are happy with how I have merged it, so it can be merged properly onto dev branch.

### CODE REVIEW

File: src/test/java/com/kainos/ea/backend/controllers/JobFamilyControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.*;
+@ExtendWith(MockitoExtension.class)
+class JobFamilyControllerTest {
+
+    @Mock
+    private JobFamilyService jobFamilyService;
+
+    @Test
+    public void whenQueryingDataJobs_expect_ServiceCalledPassback() {
```
Comment (sebastianczajkowski) [inline src/test/java/com/kainos/ea/backend/controllers/JobFamilyControllerTest.java:19]: Could you add an underscore after `when` so it is consistent with our tests? 
`when_QueryingDataJobs_expect_ServiceCalledPassback()`

File: src/test/java/com/kainos/ea/backend/services/JobFamilyServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+class JobFamilyServiceTest {
+    @Mock
+    private JobFamilyRepository jobFamilyRepository;
+
+    @Test
+    void when_getJobFamiliesByCapabilityName_expectRepositoryCalledPassback() {
```
Comment (sebastianczajkowski) [inline src/test/java/com/kainos/ea/backend/services/JobFamilyServiceTest.java:21]: Could you add an underscore after `expect` so it is consistent with our tests?
`when_getJobFamiliesByCapabilityName_expect_RepositoryCalledPassback()`

**Review by sebastianczajkowski — CHANGES_REQUESTED** [review]
I left 2 comments in the code

**Review by sebastianczajkowski — APPROVED** [review]
Looks good

**Review by michaltangri — CHANGES_REQUESTED** [review]
`@Id
    @Column(name = "jobFamilyName")
    private String jobFamilyName;`
I think this field in **JobFamily.java** should be named _name_ according to our coding conventions

**Review by michaltangri — APPROVED** [review]

**Pull request merged**